tests/Unit/Evolution/Ader/Test_Matrices.cpp

```diff
@@ -282,7 +282,12 @@ void test_predictor_inverse_temporal_matrix() {
       };
   for (size_t i = Spectral::minimum_number_of_points<
            Spectral::Basis::Legendre, Spectral::Quadrature::GaussLobatto>;
-       i <= Spectral::maximum_number_of_points<Spectral::Basis::Legendre>;
+       i <= 12;
+
+       /*The maximum number of allowed collocation points is given by
```

Comment:
Maybe: `We only derived the expected matrices up to 12 points, so we only test to that.`? Then it's clear _why_ we don't test more, not just that we don't.